TaskManager/UIColor+Extension/UIColor+Extension.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  UIColor+Extension.swift
```

Comment:
lets rename this folder to Extensions because we will have more extensions later on